src/main/java/org/broadinstitute/hellbender/engine/GATKTool.java

```diff
@@ -66,8 +67,8 @@
     @Argument(fullName = SECONDS_BETWEEN_PROGRESS_UPDATES_NAME, shortName = SECONDS_BETWEEN_PROGRESS_UPDATES_NAME, doc = "Output traversal statistics every time this many seconds elapse", optional = true, common = true)
     private double secondsBetweenProgressUpdates = ProgressMeter.DEFAULT_SECONDS_BETWEEN_UPDATES;
 
-    @Argument(fullName = StandardArgumentDefinitions.DISABLE_SEQUENCE_DICT_VALIDATION_NAME, shortName = StandardArgumentDefinitions.DISABLE_SEQUENCE_DICT_VALIDATION_NAME, doc = "If specified, do not check the sequence dictionaries from our inputs for compatibility. Use at your own risk!", optional = true, common = true)
-    private boolean disableSequenceDictionaryValidation = false;
+    @ArgumentCollection
+    SequenceDictionaryValidationArgumentCollection seqValidationArguments = new SequenceDictionaryValidationArgumentCollection.StandardValidationCollection();
```

Comment:
Shouldn't this delegate to a `getSequenceDictionaryValidationArgumentCollection()` method like the Spark one does ?

Reply:
Done, although we don't have any use case for it yet.